### ISSUE TITLE
it does not worked on Lemmy 0.19

### ISSUE THREAD

**Issue by dtantono** [CMahaff/lasim#41]
**Describe the Bug**
I suppose Since lemmy will move to 0.19 and I think this app is broken

**Information**
LASIM Version: 0.2.2
LASIM Profile Version:2
Lemmy Source Instance URL: https://lemmy.sdf.org/
Lemmy Source Instance Version: 0.19.0-rc.4
Lemmy Target Instance URL: https://lemmy.today/
Lemmy Target Instance Version: 0.18.4

**Output of LASIM**
IMPORTANT: Make sure you do not post your authentication tokens here by accident!

![image](https://github.com/CMahaff/lasim/assets/29157187/86d3abc7-1295-48c4-a520-02d0cca02b09)

**Comment by CMahaff** [comment]
Correct. I have been busy with a lot of personal life stuff and haven't updated LASIM to support 0.19 yet. I'm not home for the next week, but after that I may take a stab at it.

Though fair warning, it will probably be the last update for the app regardless - Lemmy 0.19 is supposed to feature its own account import/export system, so this app probably won't be needed anymore!

**Comment by DraconicNEO** [comment]
> Though fair warning, it will probably be the last update for the app regardless - Lemmy 0.19 is supposed to feature its own account import/export system, so this app probably won't be needed anymore!

Depending on how that new system works it might be a good idea to Keep LASIM around, for example if they make it so you can only transfer to accounts as opposed to backing up your settings locally and importing them then it would probably be good to have a tool like this, especially in the case of defederated instances where such a migration on the official system would not be possible.

**Comment by Tipoff4317** [comment]
> Though fair warning, it will probably be the last update for the app regardless - Lemmy 0.19 is supposed to feature its own account import/export system, so this app probably won't be needed anymore!

For my use case, the new native import/export seems to work well enough.  Thanks for a very useful work.  You definitely make my life more pleasant.

**Comment by MechDR** [comment]
@CMahaff It will be needed for a while, since not all instances are still on version 0.19.

**Comment by DraconicNEO** [comment]
> @CMahaff It will be needed for a while, since not all instances are still on version 0.19.

@CMahaff Maybe make it so it exports a config file which is compatible with 0.19 to be able to import it into Lemmy without having to use it on Instances which aren't compatible. Could that work as an alternative to fixing the login and connection issues?
